src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/GraphQLMiddleware.cs

```diff
@@ -122,7 +122,7 @@ private async Task ExecuteAsync(HttpContext context)
             catch (Exception e)
             {
                 await _serializer.WriteErrorAsync(context, "An error occurred while processing the GraphQL query", e);
-                return;
+                throw;
```

Comment:
This will result in a generic OC error page, as opposed to the client getting the GraphQL error produced in `WriteErrorAsync()`. The PR title is about logging the error; actually logging it here would fit that.

Reply:
done